testing/bucket_service.go

```diff
@@ -744,7 +838,18 @@ func FindBuckets(
 				filter.Name = &tt.args.name
 			}
 
-			buckets, _, err := s.FindBuckets(ctx, filter)
+			opt := platform.FindOptions{}
```

Comment:
If using the argument `findOptions` as suggested above we could get rid of this IFs.